src/consensus/params.cpp

```diff
@@ -34,6 +34,18 @@ namespace Consensus {
             .specification = "https://zips.z.cash/zip-0214",
             .valueNumerator = 8,
             .valueDenominator = 100,
+        },
+        {
+            .recipient = "Zcash Community Grants NU6",
+            .specification = "https://zips.z.cash/zip-0214",
```

Comment:
This should be the ZIP that has Alternative 2 from https://zips.z.cash/draft-nuttycom-funding-allocation